### ISSUE TITLE
--accountemail

### ISSUE THREAD

**Issue by FernandoMiguel** [Neilpang/acme.sh#806]
now that account.conf doesnt contain an email field by default,
what's the process for the account to have an email for contact and alert expire?

is it to run acme.sh --accountemail email@provider.tld 
and then acme.sh --update-account
?

**Comment by Neilpang** [comment]
yes,  you can use `--accountemail` with `--update-account` or `--install`

**Comment by sjau** [comment]
Wouldn't it be good if the --acountemail option was also mentioned on the front page instructions?

**Comment by Neilpang** [comment]
@sjau   You can add it in one of the Wiki page.  I don't want to make the front page a mess.

**Comment by FernandoMiguel** [comment]
given how important is to receive email alerts from LE (like domain is expiring cause renewal is failing) i would strongly recommend we push users to register their accounts.

@Neilpang maybe on 1st account _require_ registration with a flag to ignore it ?
and yes, i believe the front page should reference this too

**Comment by Neilpang** [comment]
@FernandoMiguel 
Actually, it was:  https://github.com/Neilpang/acme.sh#2-or-install-from-git
https://github.com/Neilpang/acme.sh/wiki/How-to-install

**Comment by Neilpang** [comment]
@FernandoMiguel   Most of the users just want a quick cert in minutes.   Nobody cares the email.  And those who cares will come back to read the doc and find it.

**Comment by FernandoMiguel** [comment]
if you believe that to be true, ok.
i'm experienced with this, and still sometimes forget to register

**Comment by Neilpang** [comment]
@FernandoMiguel 
Emmmm,  maybe, we can show a red warning message if we found the account email is never set.

what do you think ?

**Comment by FernandoMiguel** [comment]
I nice worded warning in a different color at the end of the run would be helpful, yes

**Comment by sjau** [comment]
That sounds like a good idea:

If you just run ./acme.sh --install

And don't provide an email and exisiting account.conf has no email either, then just a warning/information like this could be useful:

`NOTICE: Currently no email is set for this account. This means you won't be notified about pending certificate expirations. Run 'acme.sh --update-account --accountemail "your email address"' to add an email.`

Ok, wording can be improved :)

**Comment by neilpang** [comment]
Closing as stale: no activity for over 18 months. If this still applies on the current release, please open a new issue with the `--debug 2` log.
